### PR TITLE
New version of Nexmo Developer landing page for Hanselminutes podcast

### PR DESCRIPTION
**Pull request by martyndavies**
## Description

Added a new route (developer.nexmo.com/hansel) that listeners to the Hanselminutes podcast will be directed to. It is a version of the standard Nexmo Developer landing page, but with custom text for those particular visitors and a more obvious 'Try It Fee' button.

## Deploy Notes

Nothing special needed for this deploy.

### CODE REVIEW

File: app/views/static/podcast.html.erb [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<div class="center spacious">
+  <h1>
+      <i class="icon icon--large <%=@content['images']['top_icon']%>"></i><br><br>
+      <%=@content['headers']['leader']%>
+  </h1>
+  <br />
+  <h2 class="light"><%=@content['headers']['subheading']%></h2>
+  <br />
+  <a role="link" aria-label="Try it free" class="button" href="https://dashboard.nexmo.com/sign-up?ref=hansel-dr">Try It Free</a>
```
Comment (mheap) [inline app/views/static/podcast.html.erb]: Should `ref` come from config too?

File: app/views/static/podcast.html.erb [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+      <a name="products"></a>
+
+      <%= render 'products' %>
+      <hr>
+      <% if @content['sdk']['highlight'] %>
+      <br />
+      <div class="github-cards">
+        <div>
+          <a href="https://github.com/Nexmo/nexmo-dotnet" class="github-card" data-github="Nexmo/nexmo-dotnet">
```
Comment (mheap) [inline app/views/static/podcast.html.erb]: It feels like the library to highlight should come from the config file to me

File: app/views/static/podcast.html.erb [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      <div class="row">
+        <div class="columns small-12 medium-6">
+          <div class="center spacious content-block">
+            <h3>
+              <a href="/tools">
+                <i class="icon icon-toolbox"></i>
+                SDKs &amp; Tools
+              </a>
+            </h3>
+            <p>The Nexmo libraries allow you to get up and running with<br>Nexmo APIs quickly in your language of choice.</p>
```
Comment (mheap) [inline app/views/static/podcast.html.erb]: We can probably link to https://developer.nexmo.com/tools here

**Comment by martyndavies** [comment]
I like the XwY content idea @mheap. I'll build a new section for that and move the community section up.

**Comment by martyndavies** [comment]
I've added a section featuring two of the latest .NET related blog posts. It is currently hardcoded so that this can be live by the 27th July. I will continue to work on making it dynamic but in the interest of time, let's get this live.

**Pull request merged**